public/lsp/dbo.php

```diff
@@ -22,24 +25,24 @@
 $MAX_LOGIN_ATTEMPTS = 6;
 
 /*
- * Global paths
- */
+* Global paths
+*/
 $TMP_DIR = $_SERVER['DOCUMENT_ROOT'] . '/../tmp/';
 $DATA_DIR = $_SERVER['DOCUMENT_ROOT'] . '/../tmp/';
 $LSP_URL = '/lsp/';
 
 /*
- * By default, the LSP will use the default database values defined above
```

Comment:
The comment/styling changes seem unwarranted?

Reply:
Fixed